fedimint-core/src/util/mod.rs

```diff
@@ -319,42 +323,42 @@ pub fn handle_version_hash_command(version_hash: &str) {
 ///   error of the closure is returned
 pub async fn retry<F, Fut, T>(
     op_name: impl Into<String>,
+    statergy: impl backon::BackoffBuilder,
```

Comment:
typo

Reply:
we can live with typo :)
much better than reacquiring approvals